lib/greed/score.rb

```diff
@@ -1,44 +1,62 @@
-module Greed 
-  def calc_score(dice)
-    # Catch nil parameter
-    raise ArgumentError, "Dice must be provided" unless dice
-    count_hash = count_dice_results(dice)
-    calculate_score(count_hash)
-  end
+module Greed
+  class Score
+    def calc_score(dice)
+      # Catch nil parameter
+      raise ArgumentError, "Dice must be provided" unless dice
+      count_hash = count_dice_results(dice)
+      score = calculate_score(count_hash)
+      live_dice = count_live_dice(count_hash)
+      {'score' => score, 'live_dice' => live_dice}
```

Comment:
Returning a hash here seems really strange. What do these values have in common? What do they represent? The answer to that points to an object that you're missing or using improperly.